index.html

```diff
@@ -0,0 +1,142 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Home Page</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700&display=swap" rel="stylesheet">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@900&display=swap" rel="stylesheet">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@700&display=swap" rel="stylesheet">
+  <script src="https://kit.fontawesome.com/c90d8a2f33.js" crossorigin="anonymous"></script>
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="about.css">
+</head>
+<body>
+  <header>
+    <nav class="social-head">
+      <ul class="nav-list">
+        <li><a target="_blank" rel="noopener" href="https://www.facebook.com/uonbi.ac.ke/"><i class="fa-brands fa-facebook-square"></i></a></li>
+        <li><a target="_blank" rel="noopener" href="https://twitter.com/uonbi?ref_src=twsrc%5Egoogle%7Ctwcamp%5Eserp%7Ctwgr%5Eauthor"><i class="fa-brands fa-twitter-square"></i></a></li>
+        <li><a href="#">Logout</a></li>
+        <li><a href="#">School Page</a></li>
+        <li><a class="engo" href="#">English</a></li>
+      </ul>
+    </nav>
+
+    <nav class="navbar">
+      <div class="ham" id="ham">
+        <span class="bar"></span>
+        <span class="bar"></span>
+        <span class="bar"></span>
+      </div>
+      <a href="index.html" class="brand"><img src="./imgs/uonlogo.png" alt="uon logo"></a>
+      <ul class="navmenu" id="navmenu">
+          <li  class="items item1"><a href="about.html" class="nav-link about-link" id="about">About</a></li>
+          <li class="items item1"><a href="#programs" class="nav-link program-link">Program</a></li>
+          <li class="items item1"><a href="#guests" class="nav-link join-link">Speakers</a></li>
+          <li class="items"><a href="#sponsor" class="nav-link sponsor-link">Sponsor</a></li>
+          <li class="items"><a target="_blank" rel="noopener" href="https://www.uonbi.ac.ke/" class="nav-link school-link" id="school">UoNSchool</a></li>
+      </ul>
+
+    </nav>
+
+
+  </header>
+
+  <main>
+    <section class="first">
```

Comment:
- [ ] [OPTIONAL] This is a suggestion based on your video, you can use any image just add an overlay with the same color of the original design, and some opacity. you can do the same for your about page